vector-im/element-android
# Changes room hierarchy endpoint to stable

## Type of change


## Content



Adds stable room hierarchy endpoint with a fallback to the unstable one

## Motivation and context



Closes https://github.com/vector-im/element-android/issues/4462

## Screenshots / GIFs



N/A

## Tests



**Testing Normal Functionality**
1. Login with any account belonging to spaces, preferrably one with subspaces too
2. Open the home navigation drawer and see that you can see your spaces and subspaces as normal

**Testing Fallback**
- Set debug points in ResolveSpaceInfoTask lines 49 and 51
- Force a fail on the stable request (either via code or via a network interceptor) and see that it falls back to the unstable API

## Tested devices

- OS version(s): Android 10

## Checklist
